vividus/src/main/java/org/vividus/report/MetadataLogger.java

```diff
@@ -99,6 +110,33 @@ private static void logExecutionStatistics(Formatter message)
         message.format(row, "Skipped", story.getSkipped(), scenario.getSkipped(), step.getSkipped());
         message.format(rowsSeparator);
         message.format(row, "TOTAL", story.getTotal(), scenario.getTotal(), step.getTotal());
+        addFailureTable(message);
+    }
+
+    private static void addFailureTable(Formatter message)
+    {
+        Set<Failure> failureMessages = StatisticsStoryReporter.getFailureMessages();
+        if (failureMessages.isEmpty())
```

Comment:
I think if `statistics.print-failures=true`, but no failures are occurred we should print some message, like `No failures occurred`